docs/oauth2.md

```diff
@@ -0,0 +1,30 @@
+Oauth2
+======
+
+Oauth2 is a django app that will can be used to cache your `django-oauth-toolkit`
+access token model.
```

Comment:
I don't remember why `django-oauth-toolkit` is a dev dependency of this project, but what if we put it in [extra_require](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies)?